src/OrbitGl/CaptureWindow.cpp

```diff
@@ -392,7 +405,7 @@ std::unique_ptr<AccessibleInterface> CaptureWindow::CreateAccessibleInterface()
 void CaptureWindow::Draw() {
   ORBIT_SCOPE("CaptureWindow::Draw");
   uint64_t start_time_ns = orbit_base::CaptureTimestampNs();
-  bool time_graph_was_redrawn = false;
+  bool time_graph_was_redrawn = time_graph_ != nullptr ? time_graph_->IsRedrawNeeded() : false;
```

Comment:
bool time_graph_was_redrawn = time_graph_ != nullptr && time_graph_->IsRedrawNeeded()

Reply:
🤔 I fixed this before I moved the commit to a different branch... that's weird. Thanks for spotting this!

Reply:
Ah, this was contained in another PR... would have made sense here, but it doesn't matter too much. It's correct in main.